docs/content/installation/install_hostdefinition.md

```diff
@@ -0,0 +1,31 @@
+# Installing the `HostDefinition` custom resource
+
+Install the `HostDefinition` custom resource to enable automatic host definitions from within the CSI driver.
+
+Use the following steps to install the `HostDefinition` custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+**Note:** Only hosts added after installation are automatically defined with the CSI driver.
+
+1. Download the manifest from GitHub.
+
+        ```
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/deploy/installer/generated/host-definition.yaml > host-definition.yaml
+        ```
+
+2. Verify that the hostdefinition is in the _Ready_ phase.
+
+    ```
+    $> kubectl get hostdefinition
+    NAME                  AGE    PHASE   STORAGE          HOST
+    <k8s_resource_name1>  102m   Ready   <storage_name>   <host_name_w1>
+    k8s_resource_name2>  102m   Ready   <storage_name>   <host_name_w2>
+    ```
+
+    If in an `Error` state, a retry can be forced using TBD.
+    
+3.  (Optional) Update the prefix and connectivity fields in the `HostDefiner.yaml`.
+
+    |Field|Description|
+    |---------|--------|
+    |prefix|Adds a prefix to the hosts defined by the CSI driver.|
+    |connectivity|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `iscsi` for use with iSCSI connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `nvme` for use with NVME over Fibre Channelconnectivity (Spectrum Virtualize storage systems only)<br>By default, this field is blank and the driver selects the strongest of available connectivity options.|
```

Comment:
the outcome of the default it is not clear by `strongest`

Reply:
dominant?

Reply:
in the market?
I suggest something like the following:
```
By default, this field is blank and the driver selects the first of available connectivity types, according to the following order/hierarchy:
NVMe, FC, iSCSI.
```

Reply:
updated

Reply:
`available on the storage system` @matancarmeli7 do we check the availability on storage? don't we check the availability on the worker?

Reply:
we check it on the worker now

Reply:
updated